tests/unit_node/_fs/_fs_read_test.ts

```diff
@@ -320,3 +321,35 @@ Deno.test({
     closeSync(fd);
   },
 });
+
+Deno.test({
+  name: "accepts non Uint8Array buffer",
+  async fn() {
+    const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
+    const testData = path.resolve(moduleDir, "testdata", "hello.txt");
+    const buffer = new ArrayBuffer(1024);
```

Comment:
Should this be its own test case? I don't see it being used anywhere.

Reply:
Do you mean it's not worth having a dedicated test case for this?

If so, yeah I debated that. I'd be fine removing it